webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2367,7 +2367,10 @@ const firstVisibleMsg = {
                         'n-vis',
                         'vis'
                     );
-                    window.Applozic.ALSocket.reconnect();
+                    setTimeout(() => {
+                        console.log('reaconnecting socket');
```

Comment:
what is the guarantee that this will work? and things will happen at within time frame?
can this be event based? wait for the initialization then call this, or keep doing retry?